src/server/actions/study-job-actions.ts

```diff
@@ -7,20 +7,30 @@ import { StudyJobStatus } from '@/database/types'
 import { revalidatePath } from 'next/cache'
 import { USING_S3_STORAGE } from '@/server/config'
 import { devReadCodeFile } from '@/server/dev/code-files'
+import { attachResultsToStudyJob } from '@/server/results'
 import { siUser } from '@/server/queries'
 
-export const updateStudyJobStatusAction = async (info: MinimalJobInfo, status: StudyJobStatus) => {
+export const updateStudyJobStatusAction = async (info: MinimalJobInfo, status: StudyJobStatus, results?: string[]) => {
```

Comment:
I know you didn't come up with the name but this isn't really an update, it's  an append.   The action name at first made me afraid y'all were calling `db.updateTable`.  

The different states is also a bit confusing, maybe we should split it into 2 functions, an `approveStudyJobResults` and `rejectStudyJobResults`?

Reply:
yeah we talked about doing that while we were pairing on this - i think that makes sense to me - will update